ajax_with_jquery_exercise/script.js

```diff
@@ -0,0 +1,325 @@
+$(function() {
+
+	//let $submit = $('#login');
+	let submitToggle = false;
+
+	let $section = $("#listContent");
+	let $listSection = $("#theList");
+
+	let $login = $("#login");
+	let $favorites = $("#favorites");
+	let $logout = $("#logout");
+	$favorites.css("display",'none');
+	$logout.css("display",'none');
+
+	let authToken = localStorage.getItem("token");
+	toggleNav();
+
+	function toggleNav(){
+		console.log("toggle token", authToken);
+		if ( ! ("token" in localStorage)){
+			$login.css("display",'');
+			$favorites.css("display",'none');
+			$logout.css("display", "none");
+		}else{
+			// hide #login. because you're logged in.
+			$login.css("display",'none');
+			$favorites.css("display",'');
+			$logout.css("display", '');
+		}
+		
+	}
+	//$section.hide();
+
+	$logout.on('click',function(e){
+		// remove auth from local storage.
+		localStorage.removeItem("token");
+		authToken = undefined;
```

Comment:
this is a great use case for `null` instead of `undefined`. If we want to assign a variable to "no value" use `null`